integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/apim/utils/StoreUtils.java

```diff
@@ -64,14 +64,26 @@ public static APIKeyDTO generateAPIKey(String applicationId, String keyType, Res
 
     public static String generateUserAccessToken(String apimServiceURLHttps, String applicationId, User user,
```

Comment:
What better if we can rename this to `generateUserAccessTokenProduction`. May be by adding an overloaded method

Reply:
This will change this method in all the testcases too (because it is used in every test file). Still okay to update this?

Reply:
how about having a overload method called `generateUserAccessTokenProduction`

Reply:
Lets do these changes after alpha...

Reply:
created an issue to track this https://github.com/wso2/product-microgateway/issues/2369